sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -61,44 +62,70 @@ const deserializationContentTypes = {
   ]
 };
 
+/**
+ * Provides configuration options for AppConfigurationClient
+ */
+export interface AppConfigurationClientOptions {
+}
+
+/**
+ * Provides internal configuration options for AppConfigurationClient
+ */
+export interface InternalAppConfigurationClientOptions extends AppConfigurationClientOptions {
+  /**
+   * The sync token cache to use for this client.
+   * NOTE: this is an internal option, not for general client usage.
+   * 
+   * @internal
+   * @ignore
+   */
+  syncTokens?: SyncTokens;
+}
+
 /**
  * Client for the Azure App Configuration service.
  */
 export class AppConfigurationClient {
   private client: AppConfiguration;
   private spanner: Spanner<AppConfigurationClient>;
+  private _syncTokens: SyncTokens;
 
   /**
    * Initializes a new instance of the AppConfigurationClient class.
    * @param connectionString Connection string needed for a client to connect to Azure.
    */
-  constructor(connectionString: string);
+  constructor(connectionString: string, options?: AppConfigurationClientOptions);
```

Comment:
Constructor docs need to be updated to call out the options parameter (even if it is just a placeholder for now).